source/SquashApp.mc

```diff
@@ -1,41 +1,65 @@
+using Toybox.System;
 using Toybox.Application as App;
 using Toybox.WatchUi as Ui;
 using Toybox.Sensor as Snsr;
 
 //! Class that represents the Squash Application
 class SquashApp extends App.AppBase {
 
-	//! Object that contains the data that will
-	//! be displayed on screen
-	hidden var dataTracker;
+    //! Object that contains the data that will
+    //! be displayed on screen
+    hidden var dataTracker;
 
-	//! Constructor
+    //! Points to win a set
+    hidden var setMaxScore = 99;

```

Comment:
I agree up to a certain point. My reasons for this:
- it adds readability to the constructor, altough it may as well be achieved with local variables
- it makes prevents app-crashes (initialize variables as Int)
- it helped debugging (you may noticed the initial-values are different everywhere)

For reasons unclear to me readKeyInt() sometimes returned NULL instead of the default value. If the variable is initialized as Int it stays Int even if readKeyInt() returned NULL. At least that is what I saw when debugging with the emulator. To be on the safe side I moved the variables out of the contructor into the object and made sure they are properly initialized as last resort failback. Better to have at least some Int value there as to see the app crash horribly.